### ISSUE TITLE
Black Screen PS4 Base Model, XUbuntu

### ISSUE THREAD

**Issue by rek-age23** [thestr4ng3r/chiaki#133]
Currently attempting remote play on my laptop running Xubuntu. All I get is a black screen.

[Here is the session log](https://drive.google.com/file/d/1IwAfmv7xp-HYkbK1bPqVa4pReKsq0e2z/view?usp=drivesdk) 

Any help is appreciated

**Comment by amboxer21** [comment]
I am getting that same black screen from my Gentoo box

**Comment by amboxer21** [comment]
> Currently attempting remote play on my laptop running Xubuntu. All I get is a black screen.
> 
> [Here is the session log](https://drive.google.com/file/d/1IwAfmv7xp-HYkbK1bPqVa4pReKsq0e2z/view?usp=drivesdk)
> 
> Any help is appreciated

If you look at my post, I am willing to bet that you need to open more ports on your router like I did.

**Comment by rek-age23** [comment]
I am relatively new to Linux and changing settings on my router, and as such normally need much handholding in tutorials. I got my new router a few months ago (and have had Linux for a month now) and have done much fiddling, and my router before I didnt have the administrator login, and thus couldn't edit any settings. I would appreciate if you could make a specific tutorial to open these such ports, and not just the iptable rules as I have no idea where to even access that. I apologise for my lack of knowledge.

**Comment by amboxer21** [comment]
Yeah dude idk if this is the case for you @rek-age23. It seems like that's not the case though just from looking at your logs. In my logs when I first experienced the black screen issue, it complained about the connection being refused, so I put up a wireshark capture and saw ICMP replies from my PS4 to the originating host(my laptop) complaining that dst port 9295 was unreachable. I don't see that there in your logs. I am not an expert and could be wrong but hopefully the owner of this repo will be able to help you. I posted my solution in case someone else encounters the same issue. Sorry brother and I hope everything works out for you!
